### PR TITLE
test: Update documentation URLs used in tests

### PR DESCRIPTION
**Pull request by Parthvi12**
## Description

With the new documentation structure, updating the URLs which are being used in tests.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/client/src/constants/ThirdPartyConstants.tsx [hunk 1]
```diff
@@ -8,16 +8,16 @@ export const GITHUB_RELEASE_URL =
...
-  "https://docs.appsmith.com/setup/instance-configuration/single-sign-on-sso/openid-connect-oidc";
+  "https://docs.appsmith.com/getting-started/setup/instance-configuration/authentication/openid-connect-oidc";
 export const SAML_SIGNUP_SETUP_DOC =
-  "https://docs.appsmith.com/setup/instance-configuration/single-sign-on-sso/security-assertion-markup-language-saml";
+  "https://docs.appsmith.com/getting-started/setup/instance-configuration/authentication/security-assertion-markup-language-saml";
 export const EMAIL_SETUP_DOC =
-  "https://docs.appsmith.com/setup/instance-configuration/email";
+  "https://docs.appsmith.com/getting-started/setup/instance-configuration/email";
 export const SIGNUP_RESTRICTION_DOC =
-  "https://docs.appsmith.com/setup/instance-configuration/disable-user-signup#disable-sign-up";
```
Comment (jnikhila) [inline app/client/src/constants/ThirdPartyConstants.tsx:23]: This link is for a section on the disable-user-signup page. The updated URL will be https://docs.appsmith.com/getting-started/setup/instance-configuration/disable-user-signup#disable-sign-up

File: app/client/src/pages/Editor/gitSync/utils.ts [hunk 2]
```diff
@@ -78,9 +78,9 @@ export function changeInfoSinceLastCommit(
 }
 
 export const GIT_DOC_URLs = {
-  base: "https://docs.appsmith.com/core-concepts/version-control-with-git",
+  base: "https://docs.appsmith.com/advanced-concepts/version-control-with-git",
   import:
-    "https://docs.appsmith.com/core-concepts/version-control-with-git/connecting-to-git-repository#importing-from-a-repository",
```
Comment (jnikhila) [inline app/client/src/pages/Editor/gitSync/utils.ts:83]: There is no existing section for importing from a repository under connecting to git repository page. Should this link point to importing from a repository which is a sub page. The new url for that page will be
https://docs.appsmith.com/advanced-concepts/version-control-with-git/import-from-repository

Comment (Parthvi12) [reply app/client/src/pages/Editor/gitSync/utils.ts:83]: I have updated the URL with this link for now, @IAmAnubhavSaini Can you confirm if this is ok?

**Review by Aishwarya-U-R — APPROVED** [review]

**Review by jnikhila — COMMENTED** [review]
The URL change looks good.

**Pull request merged**